app/views/access_requests/inform_publisher.html.erb

```diff
@@ -6,15 +6,15 @@
   Inform the publisher about the change to their access:
 
   <ol class="govuk-list govuk-list--number">
-    <li>Check whether <%= link_to "they already have a DfE Sign-in account", dfe_signin_email_search(@recipient_email_address), class: "govuk-link" %></li>
-    <li>In <%= link_to "GOV.UK Notify", notify_service_link, class: 'govuk-link' %>, send an email to <strong><%= @recipient_email_address %></strong> using:
+    <li>Check whether <%= link_to "they already have a DfE Sign-in account", dfe_signin_email_search(@access_request.email_address), class: "govuk-link", data: { qa: "dfe_signin_search_link" } %></li>
+    <li>In <%= link_to "GOV.UK Notify", notify_service_link, class: 'govuk-link', data: { qa: "notify_service_link" } %>, send an email to <strong><%= @recipient_email_address %></strong> using:
```

Comment:
Is it really sufficient for us to just provide a link to Notify? Then the admin has to copy/paste or somehow manually input the email address, which is always going to be risky. Where did the decision that we implement it this way get made?

Reply:
This is the implementation as it is in the support console, as I understood it the priority was to get the functionality moved in order to decommission the support app asap and that improvements would be done after the fact.